politeiawww/cmd/cmswww/manageuser.go

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) 2017-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	cms "github.com/decred/politeia/politeiawww/api/cms/v1"
+	"github.com/decred/politeia/politeiawww/cmd/shared"
+)
+
+// ManageUserCmd allows an administrator to update Domain, ContractorType
+// and SupervisorID of a given user.
+type ManageUserCmd struct {
+	UserID             string `long:"userid" optional:"true" description:"User ID of the user to edit information"`
```

Comment:
Now that we changed how the prompts work, can we make this a required argument?  This does two things:

1. When a user uses prints the help message `cmswww -h manageuser` having the user id as a required arg communicates that it is a required field for the request.

2. When its a required arg, go-flags with print an error message if its missing so you don't have to do the validation yourself.